packages/e2e-test-utils/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## Master
```

Comment:
Wasn't `(Unreleased)` the correct way to do it? I'm not sure if there's a script replacing it with a version number. Cc @gziolo.

Reply:
https://github.com/WordPress/gutenberg/tree/master/packages#maintaining-changelogs

It's correct as is 👍

Reply:
Ah, did it change then?

Reply:
Yes, https://github.com/WordPress/gutenberg/pull/15740. We discussed it in Core JS chat and then @aduth refined it.